package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "freezeproject",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+  },
+  "dependencies": {
+    "react": "17.0.2",
+    "react-native": "0.66.4",
+    "styled-components": "^5.3.3"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.12.9",
```

Comment:
uma boa pratica é fixar a versão das dependencias (tirar o ^) para garantir que quando alguém for clonar o seu repositorio e instalar as bibliotecas, que instale as libs na versão exata em que vc testou e garantiu o funcionamento. Se deixar ccom o ^ pode acontecer de baixar uma versão mais recente e que tenha quebras de compatibilidade.

Reply:
Feito! Obrigado pela sugestão.